elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -1632,6 +1632,23 @@ public void testPostInvalidRelationship() {
                 .statusCode(HttpStatus.SC_BAD_REQUEST);
     }
 
+    @Test(priority = 42)
+    public void testSortByIdTopLevel() {
+        String sortByIdAscendingTopLevel = jsonParser.getJson("/ResourceIT/sortByTopLevelAscending.json");
+        given()
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get("/parent?sort=id")
```

Comment:
Also, I was hoping we could refactor the IT tests so that we have one for sorting, one for pagination, one for filters, etc.